<launch>
  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan_node">
    <remap from="cloud_in" to="velodyne_points"/>
	<remap from="scan" to="velodyne_scan"/>
    <param name="min_height" type="double" value="-0.10"/>
    <param name="max_height" type="double" value="0.10"/>
    <param name="angle_min" type="double" value="-3.141592"/>
    <param name="angle_max" type="double" value="3.141592"/>
    <param name="range_min" type="double" value="0.45"/>
    <param name="range_max" type="double" value="10.0"/>
    <param name="angle_increment" type="double" value="0.00287296996"/>
    <param name="target_frame" type="str" value="velodyne"/>

  </node>

</launch>
